AutomatedTesting/Gem/PythonTests/assetpipeline/fbx_tests/fbx_tests.py

```diff
@@ -758,13 +821,129 @@ def compare_scene_debug_file(asset_processor, expected_file_path, actual_file_pa
         logger.info(f"Parsing scene graph: {expected_debug_graph_path}")
         with open(expected_debug_graph_path, "r") as scene_file:
             expected_lines = scene_file.readlines()
-        assert utils.compare_lists(actual_lines, expected_lines), "Scene mismatch"
+        diff_actual, diff_expected = utils.get_differences_between_lists(actual_lines, expected_lines)
+        if diff_actual == None and diff_expected == None:
+            return None, None
+
+        # There are some differences that are currently considered warnings.
```

Comment:
nit: Consider providing cleaning up context  ie

There are some differences between platforms when comparing debug output that result in different hash values due to numeric conversion issues.

Reply:
I've added a lot more comments